sdl_android/src/main/java/com/smartdevicelink/api/ProxyBridge.java

```diff
@@ -123,7 +127,10 @@ public boolean onRPCReceived(final RPCMessage message){
 			CopyOnWriteArrayList<OnRPCListener> listeners = rpcListeners.get(id);
 			if(listeners!=null && listeners.size()>0) {
 				for (OnRPCListener listener : listeners) {
-					listener.onRpcReceived(id,message);
+					if (message instanceof RPCResponse && listener instanceof OnRPCResponseListener){
+						RPCResponse response = (RPCResponse) message;
+						((OnRPCResponseListener)listener).onResponse(response.getCorrelationID(), response);
```

Comment:
Is this necessary? I believe this should simply be the `OnRPCListener` and no modifications to this method.